### PR TITLE
HHH-13165 - Don't use confusing "this" in the User Guide admonition blocks

### PR DESCRIPTION
**Pull request by vladmihalcea**
https://hibernate.atlassian.net/browse/HHH-13165

### CODE REVIEW

File: documentation/src/main/asciidoc/userguide/chapters/caching/Caching.adoc [hunk 1]
```diff
@@ -508,8 +512,9 @@ include::{sourcedir}/SecondLevelCacheTest.java[tags=caching-statistics-example]
 
 [NOTE]
 ====
-Use of the built-in integration for https://jcp.org/en/jsr/detail?id=107[JCache] requires that the `hibernate-jcache` module jar (and all of its dependencies) are on the classpath.
-In addition a JCache implementation needs to be added as well.
+To use the built-in integration for https://jcp.org/en/jsr/detail?id=107[JCache], you need the `hibernate-jcache` module jar (and all of its dependencies) be on the classpath.
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/caching/Caching.adoc]: s/be on the/to be on the/ ?

File: documentation/src/main/asciidoc/userguide/chapters/domain/basic_types.adoc [hunk 2]
```diff
@@ -142,7 +143,8 @@ The JPA specification strictly limits the Java types that can be marked as basic
 * any other type that implements `Serializable` (JPA's "support" for `Serializable` types is to directly serialize their state to the database).
 
 If provider portability is a concern, you should stick to just these basic types.
-Note that JPA 2.1 did add the notion of a `javax.persistence.AttributeConverter` to help alleviate some of these concerns; see <<basic-jpa-convert>> for more on this topic.
+
+Note that JPA 2.1 did add the notion of a `javax.persistence.AttributeConverter` to help alleviate some of these concerns. See <<basic-jpa-convert>> for more on this topic.
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/domain/basic_types.adoc]: Not sure `did add` is necessary here, I would simply used `added`.

File: documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc [hunk 3]
```diff
@@ -195,7 +196,8 @@ include::{extrasdir}/collections-unidirectional-bag-example.sql[]
 [NOTE]
 ====
 Because both the parent and the child sides are entities, the persistence context manages each entity separately.
-Cascades can propagate an entity state transition from a parent entity to its children.
+
+The cascadeing mechanism allows you to propagate an entity state transition from a parent entity to its children.
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc]: s/cascadeing/cascading/

File: documentation/src/main/asciidoc/userguide/chapters/domain/inheritance.adoc [hunk 4]
```diff
@@ -311,7 +311,7 @@ include::{extrasdir}/entity-inheritance-table-per-class-query-example.sql[]
 ====
 Polymorphic queries require multiple UNION queries, so be aware of the performance implications of a large class hierarchy.
 
-Unfortunatelly, not all database systems support UNION ALL, so if your current Hibernate dialect does not extend any of these dialects, then UNION is going to be used instead of UNION ALL.
+Unfortunatelly, not all database systems support UNION ALL, so if your current Hibernate dialect does not extend any of these dialects, then UNION is going to be used instead of UNION ALL:
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/domain/inheritance.adoc]: s/Unfortunatelly/Unfortunately/

File: documentation/src/main/asciidoc/userguide/chapters/domain/naming.adoc [hunk 5]
```diff
@@ -93,10 +93,11 @@ would be, for example, to say that the physical column name should instead be ab
 
 [NOTE]
 ====
-It is true that the resolution to `acct_num` could have been handled in an ImplicitNamingStrategy in this case.
-But the point is separation of concerns. The PhysicalNamingStrategy will be applied regardless of whether
+It is true that the resolution to `acct_num` could have been handled using an `ImplicitNamingStrategy` in this case.
+
+But the point here is he separation of concerns. The `PhysicalNamingStrategy` will be applied regardless of whether
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/domain/naming.adoc]: s/he separation/the separation/

File: documentation/src/main/asciidoc/userguide/chapters/jdbc/Database_Access.adoc [hunk 6]
```diff
@@ -143,7 +143,7 @@ Additionally, this `ConnectionProvider` will pick up the following Hibernate-spe
 
 [IMPORTANT]
 ====
-To use this integration, the application must include the hibernate-agroal module jar (as well as its dependencies) on the classpath.
+To use the Agroal integration, the application must include the `hibernate-agroal` module jar (as well as its dependencies) on the classpath.
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/jdbc/Database_Access.adoc]: Not related to this PR but don't you think we should reorder this list of pooling technologies with the most effective first (especially having c3p0 first sounds like a bad idea).

File: documentation/src/main/asciidoc/userguide/chapters/multitenancy/MultiTenancy.adoc [hunk 7]
```diff
@@ -19,9 +19,9 @@ There are three main approaches to isolating information in these multitenant sy
 
 [NOTE]
 ====
-Each approach has pros and cons as well as specific techniques and considerations.
+Each multitenancy startegy has pros and cons as well as specific techniques and considerations.
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/multitenancy/MultiTenancy.adoc]: s/startegy/strategy/

File: documentation/src/main/asciidoc/userguide/chapters/query/hql/HQL.adoc [hunk 8]
```diff
@@ -1504,9 +1498,9 @@ include::{sourcedir}/HQLTest.java[tags=hql-entity-type-exp-example]
 
 [NOTE]
 ====
-HQL also has a legacy form of referring to an entity type, though that legacy form is considered deprecated in favor of `TYPE`.
-The legacy form would have used `p.class` in the examples rather than `type(p)`.
-It is mentioned only for completeness.
+HQL also has a legacy form of referring to an entity type using the `class` keyord, though that legacy form is considered deprecated in favor of `TYPE`.
```
Comment (gsmet) [inline documentation/src/main/asciidoc/userguide/chapters/query/hql/HQL.adoc]: s/keyord/keyword/

File: documentation/src/main/asciidoc/userguide/chapters/jdbc/Database_Access.adoc (hunk 6 above)
Comment (vladmihalcea) [reply documentation/src/main/asciidoc/userguide/chapters/jdbc/Database_Access.adoc]: c3p0 is definitely not a bad idea. It's quite a mature solution and running on hundreds of thousands of servers in production.

However, Proxool should be the last one mentioned. We could list Hikari, Vibur, and Agroal first, then c3p0 and Proxool.

Comment (gsmet) [reply documentation/src/main/asciidoc/userguide/chapters/jdbc/Database_Access.adoc]: @vladmihalcea yes that's the idea: list first the ones we recommend.

**Comment by vladmihalcea** [comment]
Pushed to 6.0 as well.

**Pull request merged**